src/sagemaker/deserializers.py

```diff
@@ -60,3 +62,34 @@ def deserialize(self, data, content_type):
             return data.read()
         finally:
             data.close()
+
+
+class CSVDeserializer(BaseDeserializer):
+    """Deserialize a stream of bytes into a list of lists."""
+
+    ACCEPT = "test/csv"
+
+    def __init__(self, encoding="UTF-8"):
+        """Initialize the string encoding.
+
+        Args:
+            encoding (str): The string encoding to use (default: "UTF-8").
+        """
+        self.encoding = encoding
+
+    def deserialize(self, data, content_type):
+        """Deserialize data from an inference endpoint into a list of lists.
+
+        Args:
+            data (object): Data to be deserialized.
```

Comment:
i wonder if there is there a stronger type that you can hint at here. this data object must respond to `.read()`, maybe BufferedIOBase or TextIOBase.

Reply:
Fixed